dockstore-webservice/src/main/java/io/dockstore/webservice/resources/MetadataResource.java

```diff
@@ -97,24 +101,30 @@
 
     private final ToolDAO toolDAO;
     private final WorkflowDAO workflowDAO;
+    private final OrganizationDAO organizationDAO;
+    private final CollectionDAO collectionDAO;
     private final DockstoreWebserviceConfiguration config;
 
     public MetadataResource(SessionFactory sessionFactory, DockstoreWebserviceConfiguration config) {
         this.toolDAO = new ToolDAO(sessionFactory);
         this.workflowDAO = new WorkflowDAO(sessionFactory);
+        this.organizationDAO = new OrganizationDAO(sessionFactory);
+        this.collectionDAO = new CollectionDAO(sessionFactory);
         this.config = config;
     }
 
     @GET
     @Timed
     @UnitOfWork(readOnly = true)
     @Path("sitemap")
-    @Operation(summary = "List all published workflow and tool paths", description = "List all published workflow and tool paths, NO authentication")
-    @ApiOperation(value = "List all published workflow and tool paths.", notes = "NO authentication")
+    @Operation(summary = "List all available workflow, tool, organization, and collection paths", description = "List all published workflow and tool paths, NO authentication")
```

Comment:
In Javadoc or the longer description, note that available means published for tools+workflows, approved for organizations and collections in approved organizations.